### ISSUE TITLE
check failure with PROJ 6 and GDAL 3

### ISSUE THREAD

**Issue by rsbivand** [MichaelChirico/geohashTools#23]
Your package fails with **rgdal** and **sf** built with PROJ 6 and GDAL 3. PROJ >= 6 will become mandatory early in 2020. See also  https://github.com/r-spatial/discuss/issues/28.
[00check.zip](https://github.com/MichaelChirico/geohashTools/files/3782389/00check.zip)

**Comment by rsbivand** [comment]
Not fixed in 0.3.0:

```
* checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(mockery)
  > library(geohashTools)
  > 
  > test_check("geohashTools")
  ── 1. Failure: gh_covering works (@test-gis-tools.R#136)  ──────────────────────
  `banjarmasin_tight` not identical to sp::`proj4string<-`(...).
  Attributes: < Component "proj4string": Attributes: < Component "comment": 1 string mismatch > >
  
  ══ testthat results  ═══════════════════════════════════════════════════════════
  [ OK: 111 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
  1. Failure: gh_covering works (@test-gis-tools.R#136) 
  
  Error: testthat unit tests failed
  Execution halted
```
If fixed in development version, please check with **rgdal** from R-Forge, PROJ > 5 and GDAL > 2, and then release.

**Comment by rsbivand** [comment]
**rgdal** will be released very soon, and your package will then have a very short grace period before being archived. Do please resolve this issue, for example using http://spatial.nhh.no/R/rgdal/rtools4-zips/ as Windows binary packages to reproduce the problems. Your use of sp::proj4string must be fully revised and adapted to the new structures discussed in https://www.r-spatial.org/r/2020/03/17/wkt.html
